tests/log-out.test.js

```diff
@@ -0,0 +1,29 @@
+import LogoutPage from '../pageobjects/logout.page'
+import LoginPage from '../pageobjects/login.page'
+
+
+describe('Banking', () => {
+    it('Should logout', async () => {
+        await LoginPage.openWeb();
```

Comment:
As you'll have to login for every test in this suite as a pre-requisite for the execution of the whole flow, you can remove that duplication by using a **beforeEach hook** and placing login related logic there. That way, you only define it once and it'll execute those steps before every logout test.